src/services/__tests__/cgnService.test.ts

```diff
@@ -14,8 +14,10 @@ const aValidSpidLevel = SpidLevelEnum["https://www.spid.gov.it/SpidL2"];
 
 const mockGetCgnStatus = jest.fn();
 const mockStartCgnActivation = jest.fn();
+const mockGetCgnActivation = jest.fn();
 
 const mockCgnAPIClient = {
```

Comment:
Why this variable was created as mockCgnAPIClient and later we have an alias of it `const api = mockCgnAPIClient;`?